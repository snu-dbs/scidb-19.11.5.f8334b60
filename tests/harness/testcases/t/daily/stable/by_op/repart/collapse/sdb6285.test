# SDB-6285.

--setup

create array DATA<value:string> [tid=0:*:0:1048576; iid=0:*:0:8; attr=0:*:0:128]
create array FIELD_MAP<sub_field_id:int64> [attribute_no=0:*:0:1024]
create array PARTICIPANT<participant_id:int64 NOT NULL> [tid=0:*:0:134217728; iid=0:*:0:16]

--test

--start-query-logging
cross_join( \
  redimension( \
    cross_join( \
      filter( \
	DATA, \
	attr > 0) as A, \
      FIELD_MAP as B, \
      A.attr, \
      B.attribute_no), \
    <value:string>[tid; iid; sub_field_id]) as C, \
  PARTICIPANT as D, \
  C.tid, D.tid, \
  C.iid, D.iid)
--stop-query-logging

--cleanup
remove(DATA)
remove(FIELD_MAP)
remove(PARTICIPANT)
