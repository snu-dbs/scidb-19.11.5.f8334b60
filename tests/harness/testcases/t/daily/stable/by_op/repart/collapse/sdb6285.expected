Query was executed successfully

Query was executed successfully

Query was executed successfully

SCIDB QUERY : <cross_join( redimension( cross_join( filter( DATA, attr > 0) as A, FIELD_MAP as B, A.attr, B.attribute_no), <value:string>[tid; iid; sub_field_id]) as C, PARTICIPANT as D, C.tid, D.tid, C.iid, D.iid)>
{tid,iid,sub_field_id} value,participant_id

Query was executed successfully

Query was executed successfully

Query was executed successfully

